pkgs/servers/pingvin-share/default.nix

```diff
@@ -17,4 +18,8 @@ recurseIntoAttrs {
   backend = callPackage ./backend.nix { inherit src version; };
 
   frontend = callPackage ./frontend.nix { inherit src version; };
+
+  passthru.tests = {
```

Comment:
This must be moved inside callPackage

Reply:
Sorry I don't understand this comment, I think I'm doing this like explained in https://github.com/NixOS/nixpkgs/tree/master/pkgs#linking-nixos-module-tests-to-a-package, what should I change?

Reply:
passthru only really makes sense when used within a mkDerivation function, the backend.nix and frontend.nix files here.